tests/tests_luks.yml

```diff
@@ -285,3 +302,9 @@
             state: absent
 
     - include_tasks: verify-role-results.yml
+
+    - name: Remove the key file
```

Comment:
Suggest using `tempfile` in a `block` and doing the removal in the `always` section e.g. https://github.com/linux-system-roles/nbde_client/blob/master/tasks/main-clevis.yml#L7

Reply:
Fixed in latest version in both `tests_luks.yml` and `tests_luks_pool.yml`.